test2code/src/main/kotlin/com/epam/drill/test2code/coverage/CoverageSender.kt

```diff
@@ -54,6 +54,10 @@ class IntervalCoverageSender(
 
     override fun stopSendingCoverage() {
         scheduledThreadPool.shutdown()
+        if (!scheduledThreadPool.awaitTermination(5, TimeUnit.SECONDS)) {
```

Comment:
I assume `awaitTermination` is required to have enough time to send coverage data.
I suggest to 
```
        if (!scheduledThreadPool.awaitTermination(5, TimeUnit.SECONDS)) {
          logger.error("Failed to send some coverage data prior to shutdown") // signal to user that some coverage data is lost
          scheduledThreadPool.shutdownNow();
        }
```
If we'll get that error message a lot - we could investigate + make termination timeout configurable.